modules/ossm-installing-multi-primary-network-mesh.adoc

```diff
@@ -141,7 +154,8 @@ $ oc --context "${CTX_CLUSTER2}" apply -n istio-system -f https://raw.githubuser
 ----
 $ istioctl create-remote-secret \
   --context="${CTX_CLUSTER2}" \
-  --name=cluster2 | \
+  --name=cluster2 \
+  --create-service-account=false | \
```

Comment:
This and the `cat <<EOF ..` are the only two things that should have been copied to this document. Everything else is specific for the primary-remote topology and should not be included here in multi-primary.